resources/translations/en.edn

```diff
@@ -27,7 +27,9 @@
                      :submission-failed "Submission failed. Check the following errors:"}
             :export-entitlements "Export entitlements as CSV"
             :handled-applications "Processed applications"
+            :invite-decider "Invite decider by email"
             :invite-member "Invite member"
+            :invite-reviewer "Invite reviewer by email"
```

Comment:
I wonder if these should rather be called `Request review by email`? From the user's point of view it does the review request.

Reply:
Originally I thought that we'd have just one Request review action and there you could choose how you want to invite by e.g. existing user autocomplete, email address list or organization reviewer autocomplete. I don't really want to see three different review actions. I think it could be made to be one action form with option on how/who to send it to?

Reply:
yeah "Request review by email" is ok, but I'll try to combine the UIs also